.github/workflows/acceptance-workflow.yml

```diff
@@ -109,6 +109,11 @@ jobs:
           name: Test Results
           path: test-*.xml
 
+      - name: Upload coverage report
+        run: bash .github/scripts/codecov-upload.sh
```

Comment:
I would probably add in the `if: ${{ !cancelled() && always() }}` conditional here as well.